programs/when.json

```
@@ -0,0 +1,10 @@
+{
+    "files": [
+        {
+            "help": "Supported natively.\n\nYou can move the file to _XDG_CONFIG_HOME/when.\n\nDo not forget, in _XDG_CONFIG_HOME/when/preferences, to add\n```\ncalendar = _XDG_CONFIG_HOME/when/calendar\n```\n",
```

```suggestion
            "help": "Supported natively.\n\nYou can move the file to _XDG_CONFIG_HOME/when_.\n\nDo not forget, in _XDG_CONFIG_HOME/when/preferences_, to add\n```\ncalendar = XDG_CONFIG_HOME/when/calendar\n```\n",
```
I'm not sure about the last one, should it maybe be `calendar = $XDG_CONFIG_HOME/when/calendar`